<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighScores</class>
 <widget class="QWidget" name="HighScores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>High Scores | Pacman</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgresources.qrc">
    <normaloff>:/img/pacman/1.png</normaloff>:/img/pacman/1.png</iconset>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/background);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="goback_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>GO BACK</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>401</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(235, 245, 238, 0.29);
border-radius: 10px;
timesTable::section { 
	background-color: rgba(235, 245, 238, 0.29); 
}</string>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="hs_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>HIGH SCORES</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../imgresources.qrc"/>
 </resources>
 <connections/>
</ui>
